mindsdb/libs/phases/data_analyzer/data_analyzer.py

```diff
@@ -330,7 +328,7 @@ class DataAnalyzer(BaseModule):
     """
 
     # @TODO get rid of scores and stats entirely
-    def compute_scores(self, col_name, sample_df, full_df, stats):
+    def compute_scores(self, col_name, sample_df, full_data_dict, stats):
```

Comment:
Is the thing that's it's receiving a dict rather and a data frame ?

Reply:
Yeah it is, so I renamed it to match